src/Symfony/Component/HttpClient/CurlHttpClient.php

```diff
@@ -74,8 +79,10 @@ public function __construct(array $defaultOptions = [], int $maxHostConnections
             return;
         }
 
-        curl_multi_setopt($mh, CURLMOPT_PUSHFUNCTION, static function ($parent, $pushed, array $requestHeaders) use ($multi) {
-            return self::handlePush($parent, $pushed, $requestHeaders, $multi);
+        $logger = &$this->logger;
```

Comment:
1. Why do you use a reference here ?
2. There is no need to create a logger variable here since we can use $this in a close since PHP 5.5 IIRC)